src/components/AutocompleteInput/AutocompleteInput.tsx

```diff
@@ -14,25 +14,30 @@ type Option = PropsWithSpread<
   HTMLProps<HTMLOptionElement>
 >;
 
-type Props = PropsWithSpread<
-  {
-    options?: (string | Option)[] | null;
-  },
-  InputProps
->;
+const isOption = (option: any): option is Option =>
+  option instanceof Object &&
+  option?.label &&
+  typeof option?.value === "string";
+
+type Props = PropsWithSpread<{ options: unknown[] }, InputProps>;
```

Comment:
This proposed change is due to the fact that this component is only used in `src/panels/AuditLogsFilterPanel/Fields/Fields.tsx`, where all the `options` property are of type `unknown[]`.